package.json

```diff
@@ -31,9 +31,7 @@
     "tt-serve-static": "bin/serve-static.js",
     "tt-wdio": "bin/wdio-runner.js"
   },
-  "browserslist": [
-    "extends browserslist-config-terra"
-  ],
+  "browserslist": [],
```

Comment:
why did we drop the browserslist-config?

Reply:
eslint was complaining about promises.  Rather than adding the polyfill config I preferred removing the browser list since this is a pure node project.